.circleci/images/primary/Dockerfile.ruby274

```diff
@@ -1,5 +1,5 @@
 # using legacy image for phantomjs
-FROM circleci/ruby:2.5.5-browsers-legacy
+FROM circleci/ruby:2.7.4-browsers-legacy
```

Comment:
Unfortunately, CircleCi never released a `browsers-legacy` image for 2.7.5, so we are limited to 2.7.4. Once we have transitioned off of the `poltergeist` gem & PhantomJS, which we use for our Cucumber suite, we *should* be able to upgrade to a next-gen `cimg/ruby:x.x.x` image.